modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -26,7 +33,11 @@ class Test{{classname}}(unittest.TestCase):
         """Test {{classname}}"""
         # FIXME: construct object with mandatory attributes with example values
         # model = {{packageName}}.models.{{classFilename}}.{{classname}}()  # noqa: E501
-        pass
+        {{#vars}}{{#required}}{{^isPrimitiveType}}{{#isModel}}{{^isEnum}}
+        imported{{dataType}} = import_module('test_' + camelToSnake("{{dataType}}"))
```

Comment:
How about adding an import at the top:
from {{packageName}} import {{classname}}
Then you do not need to use importlib
Also how about adding the model's imports to load in any classes that the parameters use like we do here:
https://github.com/spacether/openapi-generator/blob/allof_anyof_oneof_addition/modules/openapi-generator/src/main/resources/python/python-experimental/model.mustache#L26

Reply:
Done, using `__init__.py` and `import`

Reply:
Can you resolve this conversation?